tabular_export/core.py

```diff
@@ -45,7 +45,11 @@ def get_field_names_from_queryset(qs):
     field_names = []
     field_names.extend(i.target.name for i in v_qs.query.select)
     field_names.extend(v_qs.query.extra_select.keys())
-    field_names.extend(v_qs.query.aggregate_select.keys())
+    try:
+        # for django 1.10 and up (works starting in 1.8)
+        field_names.extend(v_qs.query.annotation_select.keys())
```

Comment:
Given that Django <1.8 isn't supported any more, I'm inclined to say we should just have the newer code & anyone who still needs it can use the previous release.

Reply:
Agreed 👍

Reply:
Sorry about the delay looking at this — I wasn't watching this repo for some reason